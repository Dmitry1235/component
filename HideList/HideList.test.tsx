import React from 'react';
import { HideList } from './';
import renderer from 'react-test-renderer';
import { MOCK_DATA } from './tests';

describe('HideList snapshots', () => {
  let component: any;

  beforeEach(() => {
    component = <HideList list={MOCK_DATA} />;
  });

  it('Default', () => {
    const tree = renderer.create(component);

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
