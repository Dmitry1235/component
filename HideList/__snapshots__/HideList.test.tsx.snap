// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HideList snapshots Default 1`] = `
<ul
  className="HideList"
>
  <li
    className="HideList__item"
  >
    <p
      className="Paragraph Paragraph_size_big HideList__label"
    >
      Поступило в сортировочный центр
    </p>
    <div
      className="HideList__title"
    >
      <p
        className="Paragraph Paragraph_size_medium HideList__date"
      >
        30 ноября 2020, 19:10
      </p>
      <p
        className="Paragraph Paragraph_size_medium HideList__address"
      >
        200920, Нижний Новгород
      </p>
    </div>
  </li>
  <li
    className="HideList__item HideList__item_prev-btn"
  >
    <p
      className="Paragraph Paragraph_size_big HideList__label"
    >
      Поступило в сортировочный центр
    </p>
    <div
      className="HideList__title"
    >
      <p
        className="Paragraph Paragraph_size_medium HideList__date"
      >
        30 ноября 2020, 19:10
      </p>
      <p
        className="Paragraph Paragraph_size_medium HideList__address"
      >
        200920, Нижний Новгород
      </p>
    </div>
  </li>
  <button
    className="Button Button_size_big Button_mod_custom Button_corner_primary HideList__more"
    disabled={false}
    onClick={[Function]}
    title=""
    type="button"
  >
    <div
      className="HideList__points"
    />
  </button>
  <li
    className="HideList__item"
  >
    <p
      className="Paragraph Paragraph_size_big HideList__label"
    >
      Поступило в сортировочный центр
    </p>
    <div
      className="HideList__title"
    >
      <p
        className="Paragraph Paragraph_size_medium HideList__date"
      >
        30 ноября 2020, 19:10
      </p>
      <p
        className="Paragraph Paragraph_size_medium HideList__address"
      >
        200920, Нижний Новгород
      </p>
    </div>
  </li>
  <li
    className="HideList__item"
  >
    <p
      className="Paragraph Paragraph_size_big HideList__label"
    >
      Поступило в сортировочный центр
    </p>
    <div
      className="HideList__title"
    >
      <p
        className="Paragraph Paragraph_size_medium HideList__date"
      >
        30 ноября 2020, 19:10
      </p>
      <p
        className="Paragraph Paragraph_size_medium HideList__address"
      >
        200920, Нижний Новгород
      </p>
    </div>
  </li>
</ul>
`;
